spec/Section 6 -- Execution.md

```diff
@@ -143,6 +147,88 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
     selection set.
   * Return an unordered map containing {data} and {errors}.
 
+### Subscription
+
+Unlike queries and mutations, subscriptions have a lifetime that consists of three
+phases. Between the the subscribe and unsubscribe phases, the subscription is
+considered to be in the "active" state.
+
+#### Subscribe
+
+The result of executing a subscription operation is a subscription object with
+the following capabilities:
+
+  * **Must** support observation of the associated publish stream (for example,
+  via iteration, callbacks, or reactive semantics).
+  * **Must** support cancellation of the subscription, see {Unsubscribe}.
+  * **Must** support a way to identify the subscriber/subscription pair (for
+    example, a GUID/callback table or closure over the callback).
+  * **May** include an initial response associated with executing the selection
+  set defined on the subscription operation.
+
+Subscribe(schema, subscription, operationName, variableValues, initialValue):
+
+  * Let {subscriptionType} be the root Subscription type in {schema}.
+  * Assert: {subscriptionType} is an Object type.
+  * Let {selectionSet} be the top level Selection Set in {subscription}.
+  * Let {rootField} be the first top level field in {selectionSet}.
+  * Let {eventStream} be the result of running {MapSubscriptionEvents(rootField, variableValues)}.
+  * Optionally: let {data} be the result of running {ExecuteRequest(schema, document, operationName, variableValues, initialValue)}, and return {data} on the subscription object.
```

Comment:
Agreed. Upon further discussion with @laneyk and @leebyron, initial value behavior can be supported out-of-the-box using two sub-fields in a subscription, as in @rmosolgo's example. However, the model of an "atomic query/subscribe" is probably better supported via batching. 

To be clear, the resulting edit is that we will simply not mention server behavior with respect to initial value. If you choose to implement/use it, that does not make your server violate the spec.